webui/react/src/pages/ExperimentDetails/ExperimentDetails.tsx

```diff
@@ -28,6 +29,8 @@ type Params = {
 export const INVALID_ID_MESSAGE = 'Invalid Experiment ID';
 export const ERROR_MESSAGE = 'Unable to fetch Experiment';
 
+const settingsLoadedObs = userSettings.getAll().select((l) => l.isLoaded);
```

Comment:
q: what is the reasoning for checking if settings are loaded, given that it doesn't seem like we otherwise aren't using settings?

Reply:
the feature flag state is based on settings as well -- when the user has `f_flat_runs` disabled, but the flag is default on, the redirect fires before settings loads.